src/solids4FoamModels/solidModels/solidModel/solidModel.C

```diff
@@ -111,10 +111,11 @@ void Foam::solidModel::makeDualMesh() const
     // Create one dual face between cell centres and face centres
     const bool splitAllFaces = true;
 
-    // Hard-coded settings
+    // Hard-coded settings --> these can not be added to dictionary and returned in output-dict, because
+    // this function is qualified as 'const'. This is, even though dictionary is mutable. (Solutions?)
```

Comment:
What does this comment refer to? Do we need to do something here?

Reply:
There are default values following after this comment. These are readOrDefault from 'solidProperties'. Since my goal was to write out all default values after a successful run, I'd like to find a way to write them into the dict. However the readOrAddDefault function cannot be used in a function that is const qualified. So the commend is a question for solutions. Otherwise we could neglect these default values for now, as I had to do with values that were read from 'time().controlDict()'.

Reply:
OK, for now, I guess these values can be skipped.